tests/tests.rs

```diff
@@ -44,3 +44,14 @@ fn test_match_wildcard() {
         4,
     );
 }
+
+#[test]
+fn test_match_only_wildcard() {
```

Comment:
It is better to note the URL of this PR: https://github.com/daac-tools/trie-match/pull/4